### DISCUSSION TITLE
How to pass arguments ?

### DISCUSSION THREAD

**Discussion by caiosecco** [SuRGeoNix/Flyleaf#515] [General]
Hello All,

I looking for a C# player that can reproduce RTSP stream without delay. VLC can't do that (the best is ~500ms)
FFMpeg can, but only with custom args, including **-vf setpts=0** How to achive it using Flyleaf?

Is there anyway to pass raw arguments?

**Comment by SuRGeoNix** [comment]
Hi @workjoule, check this and let me know if you need more help https://github.com/SuRGeoNix/Flyleaf/issues/388

mainly you need Config.Player.MaxLatency = TimeSpan.FromMilliseconds(500).Ticks; but if you go lower you might have issues

**Reply by caiosecco** [reply]
That worked! 

Thanks @SuRGeoNix
